go.mod

```diff
@@ -19,6 +16,12 @@ require (
 	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42
 )
 
+require (
```

Comment:
I see no need to maintain two separate `require` sections at the top, let's merge these two.